### PR TITLE
adding vpx-batman66 to main

### PR DESCRIPTION
**Pull request by Officially-Unofficial**
Please ensure your PR includes the following:

- README.md
- launcher.png
- launcher.xml
- pinmame/cfg, pinmame/ini, pinmame/nvram, pinmame/roms folders. If they are empty, create a .gitkeep file to ensure the folders get created in the repo.
- any game specific ini or vbs files named exactly the same as the publicly available files from either VPForums or VPUniverse

Please ensure your PR **DOES NOT** include any of the following:

- VPX files
- ROMs
- PUP Media files
- Any sort of licensing text that goes against the licensing of this repository

By submitting this PR, you are agreeing that any artwork submitted is your own, or you have the legal right to distribute such artwork.

### CODE REVIEW

File: external/vpx-batman66/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+| Playfield | Controls | Backglass | DMD | ROM Required | FPS | 
+|-----------|----------|-----------|-----|--------------|-----|
+| :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | 30 |
+
+## Instructions
+
+- Copy the contents of this repo folder to your USB drive
+- Add your personalized launcher.elf and rename it to vpx-nascar.elf
+- Download the table and directb2s versions listed above and copy them into this folder
+- You're off to the races 🏁
```
Comment (n-i-x) [inline external/vpx-batman66/README.md:37]: Looks like this was copied from NASCAR. Change the witty instruction.

File: external/vpx-batman66/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Author: [hauntfreaks](https://vpuniverse.com/profile/5216-hauntfreaks/) 
+Version: 1.0
+Download: [VP Universe] https://vpuniverse.com/files/file/6949-batman-66-stern-2016-b2s/
+
+ROM
+
+Download: Included with table download
+
+PUP Pack
```
Comment (n-i-x) [inline external/vpx-batman66/README.md:19]: Remove if it's not working

**Review by n-i-x — CHANGES_REQUESTED** [review]

File: external/vpx-batman66/README.md (hunk 1 above)
Comment (Officially-Unofficial) [reply external/vpx-batman66/README.md:37]: Resolved

File: external/vpx-batman66/README.md (hunk 2 above)
Comment (Officially-Unofficial) [reply external/vpx-batman66/README.md:19]: Removed

**Pull request closed without merging**